epsagon/event.py

```diff
@@ -105,3 +107,5 @@ def set_exception(self, exception, traceback_data):
         self.exception['message'] = str(exception)
         self.exception['traceback'] = traceback_data
         self.exception['time'] = time.time()
+        if not handled:
+            self.exception.setdefault('additional_data', {})['unhandled'] = True
```

Comment:
I think it's more straight forward to save `{'handled': True}` so that the trace will be not changed in the most common event of unhandled exception.